samples/official/ai-vision-devkit-get-started/deployment.template.json

```diff
@@ -31,9 +31,10 @@
             "status": "running",
             "restartPolicy": "always",
             "settings": {
-              "image": "mcr.microsoft.com/azureiotedge-hub:1.0",
+              "image": "mcr.microsoft.com/azureiotedge-hub:1.0.4",
               "createOptions": {
                 "HostConfig": {
+                  "Dns": ["8.8.8.8"],
```

Comment:
here we are following option 1 https://docs.microsoft.com/en-us/azure/iot-edge/troubleshoot#resolution-7
 and adding daemon.json during intial out of box setup. As in corp network behind firewall DNS is different then 8.8.8.8 eg. for MSFTGUEST it is 1.1.1.1